hack/gatewayapi-conformance.sh

```diff
@@ -47,7 +47,7 @@ go mod vendor
 # because the AWS ELB needs extra ~60s for DNS propagation
 sed -i "s/MaxTimeToConsistency:              30/MaxTimeToConsistency:              90/g" conformance/utils/config/timeout.go
```

Comment:
I noticed that one more space is inserted in release-1.1 and later, the workaround could be updated
```suggestion
sed -i "s/MaxTimeToConsistency:.*30/MaxTimeToConsistency: 90/g" conformance/utils/config/timeout.go
gofmt -s -w conformance/utils/config/timeout.go
```

Reply:
Thanks!  Fixed in https://github.com/openshift/cluster-ingress-operator/pull/1152/commits/26a3918c7374cc16eac04b87c9e4267591c304a5.